### PR TITLE
Changed ingresses and networkpolicies apiGroup

### PR DESCRIPTION
**Pull request by bartsmykla**
It was changed in `infra/gcp/namespaces/namespace-user-role.yml`. It was
`extensions` before so I changed it to `networking.k8s.io`

/cc @ameukam 

Signed-off-by: Bart Smykla <bsmykla@vmware.com>

### CODE REVIEW

File: infra/gcp/namespaces/namespace-user-role.yml [hunk 1]
```diff
@@ -18,13 +18,13 @@ rules:
     resources: ["cronjobs", "jobs"]
     verbs: ["*"]
   - apiGroups: ["extensions"]
```
Comment (ameukam) [inline infra/gcp/namespaces/namespace-user-role.yml:20]: We could delete this API group. @thockin @spiffxp WDYT ?

**Comment by ameukam** [comment]
/lgtm
/hold
for comments.

**Comment by bartsmykla** [comment]
@spiffxp what is your opinion on @ameukam's question?

File: infra/gcp/namespaces/namespace-user-role.yml (hunk 1 above)
Comment (spiffxp) [reply infra/gcp/namespaces/namespace-user-role.yml:20]: Yeah, I think we should, once we've migrated everything away from it

File: infra/gcp/namespaces/namespace-user-role.yml [hunk 2]
```diff
@@ -18,13 +18,13 @@ rules:
...
-    resources: ["deployments", "ingresses", "networkpolicies"]
```
Comment (spiffxp) [inline infra/gcp/namespaces/namespace-user-role.yml:21]: I think this should be done in a followup PR

**Review by spiffxp — CHANGES_REQUESTED** [review]
I don't think we should remove access to the deprecated extensions group (and resources) until _after_ we've migrated things away from it.  So for this PR I would probably just add access to networking.k8s.io/ingresses

Is anyone interested in doing the actual migration?

**Comment by spiffxp** [comment]
We'll want this done before the cluster is migrated to v1.16

**Comment by ameukam** [comment]
/assign
I'll work on that next week.

**Comment by spiffxp** [comment]
/close
In favor of #1629

**Pull request closed without merging**